models/DataObject/ClassDefinition/Data/QuantityValue.php

```diff
@@ -167,7 +167,7 @@ public function getDefaultValue()
      */
     public function setDefaultValue($defaultValue)
     {
-        if ($defaultValue !== null) {
+        if (!empty($this->defaultValue)) {
```

Comment:
```suggestion
        if (!empty($defaultValue)) {
```

Reply:
@dvesh3 I did what you wanted, but I am unsure if that is *actually* correct. Can you describe why?